src/conditionals/programs_with_decision_points/hard/q9/DrivingFines.java

```diff
@@ -0,0 +1,62 @@
+package conditionals.programs_with_decision_points.hard.q9;
+
+/*
+ * Write a Java Program that determines the fine of a person caught speeding by asking the user how much above the
+ * speed limit is the driver, and if they were not in a school zone. If a person is driving 1 - 10 km/h above the speed
+ * limit, their fine is $50. If a person is driving 11 - 30 km/h above the speed limit, their fine is $75. If a
+ * person is driving 31 - 50 km/h above the speed limit, their fine is $100. If a person is driving 51 km/h above
+ * the speed limit, their fine is $150. If a person is in a school zone, then their fine is doubled. Make sure to
+ * print “Unknown inputs” if the input is invalid.
+ * 
+ * Sample Output:
+ *    How fast were you above the speed limit?
+ *    10
+ *    Were you in a school zone? (Enter 1 for True, and 0 for False)
+ *    1
+ *    Your fine is $100!
+ * 
+ * Distractors:
+ *    boolean speed = input.nextInt();
+ *    int char = temp.charAt(0);
+ *    if ( speed > 51 )
+ *    else if( speed>31 && speed < 50){
+ *    else if( speed>11 && speed < 30){
+ *    else if( speed>1 && speed < 10){
+ *
+ */
+
+import java.util.Scanner;
+
+public class DrivingFines {
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    System.out.println("How fast were you above the speed limit?");
+    int speed = input.nextInt();
+    System.out.println("Were you in a school zone? (Enter 1 for True, and 0 for False)");
+    int temp = input.nextInt();
+    if (temp != 1 && temp != 0) {
```

Comment:
Also I think it would be good to use nextBoolean and just have the input be true/false instead of 0 and 1